test/IRGen/big_types_corner_cases.swift

```diff
@@ -1,6 +1,7 @@
 // RUN: %target-swift-frontend -assume-parsing-unqualified-ownership-sil -enable-large-loadable-types %s -emit-ir | %FileCheck %s --check-prefix=CHECK --check-prefix=CHECK-%target-ptrsize
 // REQUIRES: optimized_stdlib
 // UNSUPPORTED: resilient_stdlib
+// UNSUPPORTED: CPU=powerpc64le
```

Comment:
Why is this unsupported on PPC64le?  Seems like its just disabling the test until it is fixed?  Can you file a SR for this?

Reply:
@compnerd I think it's a good suggestion to mark big_types_corner_cases.swift as unsupported on ppc64le till it is analysed and fixed. I can certainly file a SR for this. However I couldn't find procedure to file SR in the contribution documents (https://github.com/apple/swift and https://swift.org/contributing/). Could you please direct me to a correct procedure to file SR? Thanks!

Reply:
@asowani the issues are tracked at http://bugs.swift.org ... you should be able to register and then file the SR there.

Reply:
@compnerd Thanks for the pointer. I have created a new SR at https://bugs.swift.org/browse/SR-6953

Reply:
@compnerd I have reverted changes in test/IRGen/big_types_corner_cases.sil as they were redundant. With this, is it possible to proceed with merging code changes in remaining files? Thanks!